target/linux/ramips/base-files/lib/ramips.sh

```diff
@@ -169,6 +169,9 @@ ramips_board_detect() {
 	*"DWR-512 B")
 		name="dwr-512-b"
 		;;
+	*"DWR-921 C1")
```

Comment:
Do You test generic board detection?

Reply:
I prefer to be coherent with the format used in the file.

Reply:
@fid0did0 You should really consider @CHKDSK88 input, as I've done a full review of his PR recently.

Drop this line to use the generic board detection. Which means, use `dlink,dwr-921-c1` as userspace boardname. Instead of sticking to an outdated way of doing something, you should check the commit history to see how it was done for _all_ recent ramips additions.